scripts/west-commands.yml

```diff
@@ -0,0 +1,17 @@
+west-commands:
```

Comment:
Would it be better to have:
`scripts/west/west-commands.yml`
and then:
`scripts/west/commands/*.py`
?

Reply:
I made a change from when we initially discussed this feature: the extension commands can import west modules, but we no longer make any requirements on the file system layout.

See this comment: https://github.com/zephyrproject-rtos/west/blob/master/src/west/commands/command.py#L213

The reason I chose this layout was because "flat is better than nested" in the zen of python and there is no need to nest.

OK for you?

Reply:
That would be my preference, yes:
```
scripts/west/west-commands.yml
scripts/west/commands/*.py
```
@tejlmand can you comment on your preference as well to get more input?

Reply:
I just want to get this done and don't want to argue.

But I think having a `scripts/west` is dangerous: it might suggest that's where west lives. People will need time to understand that it's in its own repository.

But if that is the consensus I will make this change.

Reply:
i'm really fine with both ways as long as the python scripts them-selves is in sub-folder.

Reply:
Ok, I buy the arguments. Let's leave this as-is